packages/ui-library/src/elements/toast/docs/description.md

```diff
@@ -0,0 +1 @@
+`Toast.Description` can be used to add a description to the toast's content. It can be a list of strings.
```

Comment:
And in this doc maybe also mention the paragraph option;

The `Toast.Description` subcomponent allows you to add detailed content to the toast, either as a paragraph or a list of strings.